examples/metrics/src/main/java/com/google/cloud/opentelemetry/example/metrics/MetricsExporterExample.java

```diff
@@ -0,0 +1,71 @@
+package com.google.cloud.opentelemetry.example.metrics;
+
+import static java.util.Collections.singleton;
+
+import com.google.cloud.opentelemetry.metric.MetricConfiguration;
+import com.google.cloud.opentelemetry.metric.MetricExporter;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.metrics.LongUpDownCounter;
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.metrics.export.IntervalMetricReader;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Random;
+
+public class MetricsExporterExample {
+  private static final Meter METER =
+      OpenTelemetry.getGlobalMeter("instrumentation-library-name", "semver:1.0.0");
+  private static io.opentelemetry.sdk.metrics.export.MetricExporter metricExporter;
+  private static IntervalMetricReader intervalMetricReader;
+
+  private static void setupMetricExporter() {
+    try {
+      MetricConfiguration configuration =
+          MetricConfiguration.builder().setDeadline(Duration.ofMillis(30000)).build();
+      metricExporter = MetricExporter.createWithConfiguration(configuration);
+      intervalMetricReader =
+          IntervalMetricReader.builder()
+              .setExportIntervalMillis(500)
+              .setMetricExporter(metricExporter)
+              .setMetricProducers(
+                  singleton(OpenTelemetrySdk.getGlobalMeterProvider().getMetricProducer()))
+              .build();
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+  }
+
+  private static void myUseCase() {
+    LongUpDownCounter counter =
+        METER
+            .longUpDownCounterBuilder("processed_jobs")
+            .setDescription("Processed jobs")
+            .setUnit("1")
+            .build();
+    doWork(counter);
+  }
+
+  private static void doWork(LongUpDownCounter counter) {
+    Random random = new Random();
```

Comment:
nit: It's better to instantiate random once per program, i.e. make it a `private static final` field.